README.md

```diff
@@ -59,6 +59,19 @@ To enable it, add the following lines to your Jekyll site:
 
 You can find out more about Disqus' shortnames [here](https://help.disqus.com/customer/portal/articles/466208).
 
+Note that you should have set the `url` property to make it work properly.
+
+Be sure to have registered your disqus account using a good domain name like :
+
+```
+user.github.io
+```
+
+and not
+```
+https://user.github.io
+```
+
```

Comment:
Not sure if domain-name syntax needs to be documented here as its already covered in disqus' documentation. But since you've already included a few examples, you might as well mention that `www.mydomain.com` is a wrong entry as well.